src/libpriv/rpmostree-postprocess.c

```diff
@@ -63,24 +63,24 @@ run_bwrap_mutably (int           rootfs_fd,
                    GCancellable *cancellable,
                    GError      **error)
 {
-  g_autoptr(RpmOstreeBwrap) bwrap = NULL;
-  if (unified_core_mode)
-    bwrap = rpmostree_bwrap_new (rootfs_fd,
-                                 RPMOSTREE_BWRAP_MUTATE_ROFILES,
-                                 error,
-                                 "--ro-bind", "./var", "/var",
-                                 NULL);
-  else
-    bwrap = rpmostree_bwrap_new (rootfs_fd,
-                                 RPMOSTREE_BWRAP_MUTATE_FREELY,
-                                 error,
-                                 "--bind", "var", "/var",
-                                 "--bind", "usr/etc", "/etc",
-                                 NULL);
-
+  g_autoptr(RpmOstreeBwrap) bwrap =
+    rpmostree_bwrap_new (rootfs_fd,
+                         unified_core_mode ? RPMOSTREE_BWRAP_MUTATE_ROFILES :
+                         RPMOSTREE_BWRAP_MUTATE_FREELY,
```

Comment:
Minor: I'd rather see it all on one line. Otherwise it looks like a separate parameter. Or maybe something like:

```c
unified_core_mode ? RPMOSTREE_....
                  : RPMOSTREE_....,
```

We may be doing it this way elsewhere too.